hydradx/other tests/test_exploits.py

```diff
@@ -0,0 +1,645 @@
+import copy
+import math
+
+from hypothesis import given, strategies as st, settings
+from mpmath import mp, mpf
+
+from hydradx.model.amm import omnipool_amm as oamm
+from hydradx.model.amm.agents import Agent
+from hydradx.model.amm.global_state import GlobalState
+from hydradx.model.amm.trade_strategies import constant_swaps, omnipool_arbitrage
+from hydradx.tests.strategies_omnipool import omnipool_reasonable_config, omnipool_config
+mp.dps = 50
+
+# @settings(max_examples=1)
+@given(
+    st.floats(min_value=0, max_value=0.10, exclude_min=True),
+    st.floats(min_value=0, max_value=0.01, exclude_min=True),
+    # st.floats(min_value=0.90, max_value=1.1)
+)
+def test_add_liquidity_exploit(lp_multiplier, trade_mult):
```

Comment:
remind me why we moved the exploit tests out of integration tests?

Reply:
I put them back